sous-chef/sous_chef/date/get_due_date.py

```diff
@@ -5,8 +5,9 @@
 from pytz import timezone
 from sous_chef.abstract.extended_enum import ExtendedEnum, ExtendedIntEnum
 
+DEFAULT_TIMEZONE = timezone("UTC")
+
 
-# TODO make configurable?
```

Comment:
removed these todos as we haven't done it & seems unlikely others will use the repo until we have a circular product.